src/test/java/persistence/sql/ddl/PersonTest.java

```diff
@@ -0,0 +1,84 @@
+package persistence.sql.ddl;
+
+import jakarta.persistence.*;
+import org.jetbrains.annotations.NotNull;
+import org.junit.jupiter.api.Test;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import java.lang.reflect.Field;
+import java.util.Arrays;
+
+public class PersonTest {
+    Class<Person> personClass = Person.class;
+    private static final Logger logger = LoggerFactory.getLogger(PersonTest.class);
+
+    @Test
+    void getTableName() {
+        logger.debug(String.valueOf(personClass.getAnnotation(Table.class).name()));
+    }
```

Comment:
```suggestion
    @Test
    @DisplayName("테이블의 이름을 가져온다.")
    void getTableName() {
        String tableName = String.valueOf(personClass.getAnnotation(Table.class).name());
        assertThat(tableName).isEqualTo("Person");
```

1. `@DisplayName` 어노테이션을 활용하면 더 가독성 있는 테스트 코드를 작성 할 수 있을 것 같습니다 :)
2. 테스트를 검증할 때는 assertj 를 사용해주시면 됩니다!